penny_university_frontend/src/components/buttons/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faPlus, faTimes, faPen, faSave, faHeart,
+} from '@fortawesome/free-solid-svg-icons'
+import classNames from 'classnames/dedupe'
```

Comment:
Seems like we have this library as a sub dependency. I worked on this along side #179 and it is explicitly added in that PR